app/styles/ui/dialogs/_release-notes.scss

```diff
@@ -9,7 +9,7 @@
   }
 
   .dialog-header {
-    height: 100px;
+    height: 110px;
```

Comment:
Without the bump to the `height`, the upper half of that dialog looks a bit "compressed" (for lack of a better term). The margins of the close button don't look right, and one can argue that the link is too close to the divider.
<img width="679" alt="x" src="https://user-images.githubusercontent.com/3772093/50120509-97ba2c80-020a-11e9-8a00-5d536d84421c.png">

Let me know if there's a better way to handle that instead of just bumping the `height`.